Multisig/Logic/AuthenticationController.swift

```diff
@@ -39,19 +39,37 @@ class AuthenticationController {
     ///
     /// - Parameter plaintextPasscode: unsecured, "as-is" passcode
     func createPasscode(plaintextPasscode: String) throws {
-        for user in accessService.userRepository.users() {
-            accessService.userRepository.delete(userID: user.id)
-        }
-        let password = derivedKey(from: plaintextPasscode)
-        try accessService.registerUser(password: password)
-        AppSettings.passcodeWasSetAtLeastOnce = true
+        if AppConfiguration.FeatureToggles.securityCenter {
+            let password = derivedKey(from: plaintextPasscode)
+            SecurityCenter.shared.changePasscode(new: password, useBiometry: AppSettings.securityLockMethod != .passcode) { error in
+                if let error = error {
+                    LogService.shared.error("Error creating password: \(error)")
+                } else {
+                    AppSettings.securityLockEnabled = true
+                    AppSettings.passcodeWasSetAtLeastOnce = true
+                    AppSettings.passcodeOptions = [.useForLogin, .useForConfirmation]
```

Comment:
I see this part is duplicated here and in the else case, can we extract to a shared method?

Reply:
After we complete implementation, we will remove the toggle and delete everything that's in another branch. Because of this, this code duplication is not a functional duplication, i.e. we will delete the other copy.